### DISCUSSION TITLE
How to apply styles to SwiperSlide

### DISCUSSION THREAD

**Discussion by ghost** [nolimits4web/swiper#5608] [Q&A]
I can't give it a className, what's the best way to apply a style to SwiperSlide in React?

https://github.com/nolimits4web/swiper/issues/3934

**Comment by Arhell** [comment]
You can give the slider a wrapper with your own class and use inheritance to do what you need.

**Reply by ghost** [reply]
Inheritance doesn't seem to be recommended in React, is that the best bet?

**Reply by Arhell** [reply]
Perhaps, but you have a task, and what I wrote above is one of the solutions to your problem.

**Comment by ghost** [comment]
Not being able to give a className to SwiperSlide seems to be a problem with React18
https://github.com/nolimits4web/swiper/issues/5613
